README.md

```diff
@@ -169,8 +169,9 @@ Example: -r .jobscript.FastQC.*
 
 Please note that depending on your favorite shell, you may need quote your regex. 
 
-### Provide a file with several QBiC IDs
-In order to download datasets from several samples at once, you can provide a simple text file with multiple, line-separated, QBiC IDs and hand it to postman with the `-f` option.
+### Provide a manifest file with several QBiC IDs
```

Comment:
Please be careful with section headings. Changing this text breaks all links to the section. I would argue we do not need to change the heading here.
```suggestion
### Provide a file with several QBiC IDs
```